app/package.json

```diff
@@ -12,6 +12,8 @@
   "resolutions": {
     "@codemirror/state": "~6.4.1",
     "@codemirror/view": "~6.28.3",
+    "@jupyter/react-components": "^0.16.6",
+    "@jupyter/web-components": "^0.16.6",
```

Comment:
Maybe we can move the `@jupyter/ydoc` line below next to these new packages, so it's easier to notice (and somewhat sorted) while looking at the resolutions?

Reply:
Good catch, I moved the new entries instead to follow the ASCII order.